src/parser/expression.js

```diff
@@ -385,6 +388,16 @@ pp.parseExprAtom = function (refShorthandDefaultPos) {
       }
       return this.finishNode(node, "Super");
 
+    case tt._import:
+      if (!this.hasPlugin("dynamicImport")) this.unexpected();
+
+      node = this.startNode();
+      this.next();
+      if (!this.match(tt.parenL)) {
+        this.unexpected();
```

Comment:
Actually we can use https://github.com/babel/babylon/pull/172

to do `this.unexpected(null, tt.parenL);`